packages-presets/preset-web-fonts/src/types.ts

```diff
@@ -20,6 +20,11 @@ export interface WebFontMeta {
    */
   italic?: boolean // ital axis
 
+  /**
+   * Font width axis
+   */
+  width?: string | number // wdth axis
```

Comment:
```suggestion
  width?: (string | number)[] // wdth axis
```
Is its type an array?

Reply:
yes, array is better.